backends/platform/sdl/macosx/macosx.h

```diff
@@ -48,7 +48,7 @@ class OSystem_MacOSX : public OSystem_POSIX {
 	GraphicsManagerType getDefaultGraphicsManager() const override;
 #endif
 
-	//Screenshots
```

Comment:
Why remove it? Call it `// Default paths`

Reply:
Done.